src/read_utils.jl

```diff
@@ -2,6 +2,37 @@ using DelimitedFiles
 
 # ---------------------------------------------------------------------------------------- #
 
+function make_symmetryvector(bandirsd::Dict{String, Vector{Pair{UnitRange{Int64}, Vector{Int64}}}}, 
+lgirsd::Dict{String, Vector{LGIrrep{2}}}, sgnum::Integer, dim::Integer=2, has_tr::Bool = true)
```

Comment:
Should not be specialized to 2D:
```suggestion
   lgirsd::Dict{String, Vector{LGIrrep{D}}}, timereversal::Bool = true) where D
```
Other things:
- We need to give `sgnum` as input (could be inferred from any element of `lgirsd`).
- Elsewhere in this code, we call the `has_tr` kwarg `timereversal`.